<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-N1QJ0PMG" ignored="0" name="Login Page Test" tests="2" failures="2" timestamp="28 Jan 2023 13:20:50 GMT" time="25.907" errors="0">
  <testcase name="validMobileNoAndPasswordTest" time="25.811" classname="com.mwbtech.PagarPlusApp.LoginScreenTest">
    <failure type="io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException" message="The local appium server has not been started. The given Node.js executable: C:\Program Files\nodejs\node.exe Arguments: [C:\Users\Admin\AppData\Roaming\npm\node_modules\appium\build\lib\main.js, --port, 4723, --address, 0.0.0.0] 
Process output: [35m[Appium][39m Welcome to Appium v2.0.0-beta.47
[35m[Appium][39m Attempting to load driver uiautomator2...
[debug] [35m[Appium][39m Requiring driver at C:\Users\Admin\.appium\node_modules\appium-uiautomator2-driver

">
      <![CDATA[io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException: The local appium server has not been started. The given Node.js executable: C:\Program Files\nodejs\node.exe Arguments: [C:\Users\Admin\AppData\Roaming\npm\node_modules\appium\build\lib\main.js, --port, 4723, --address, 0.0.0.0] 
Process output: [35m[Appium][39m Welcome to Appium v2.0.0-beta.47
[35m[Appium][39m Attempting to load driver uiautomator2...
[debug] [35m[Appium][39m Requiring driver at C:\Users\Admin\.appium\node_modules\appium-uiautomator2-driver


at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:187)
at com.mwbtech.PagarPlusApp.BaseTest.ConfigureAppium(BaseTest.java:25)
at com.mwbtech.PagarPlusApp.LoginScreenTest.validMobileNoAndPasswordTest(LoginScreenTest.java:16)
Caused by: org.openqa.selenium.net.UrlChecker$TimeoutException: Timed out waiting for [http://127.0.0.1:4723/status] to be available after 20032 ms
at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:89)
at io.appium.java_client.service.local.AppiumDriverLocalService.ping(AppiumDriverLocalService.java:152)
at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:174)
... 27 more
Caused by: java.util.concurrent.TimeoutException
at java.util.concurrent.FutureTask.get(Unknown Source)
at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:87)
... 29 more
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- validMobileNoAndPasswordTest -->
  <testcase name="validMobileNoAndPasswordTest" time="25.867" classname="com.mwbtech.PagarPlusApp.LoginScreenTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.wbtech.PagarPlus.utils.AppiumUtils.getScreenShotPath(AppiumUtils.java:29)
at com.mwbtech.PagarPlusApp.TestUtils.Listeners.onTestFailure(Listeners.java:45)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- validMobileNoAndPasswordTest -->
</testsuite> <!-- Login Page Test -->
